tests/count_last_changes.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+# ====
+# Check that the last changes increment correctly the number of updates to the last release
+# in the info.json file
+# ====
+# Get the last changes folder
+latest=$(ls updates| sort -n | tail -n1)
+# Count the number of lines in the updates files
+lines=$(sudo wc -l updates/$latest/updates.sql | cut -d' ' -f1)
```

Comment:
you should definitely not need `sudo` for a wc

Reply:
I don't know why before it was failing without sudo but it's true that is not needed. Updated